adafruit_bme680.py

```diff
@@ -542,7 +564,7 @@ def _read(self, register, length):
                 print("\t$%02X => %s" % (register, [hex(i) for i in result]))
             return result
 
-    def _write(self, register, values):
+    def _write(self, register: int, values: int):
```

Comment:
Looks like the typing is the same as the others:

```suggestion
    def _write(self, register: int, values: ReadableBuffer) -> None:
```